node/runtime/src/lib.rs

```diff
@@ -932,9 +955,27 @@ static EVM_CONFIG: EvmConfig = EvmConfig {
 	has_ext_code_hash: true,
 	estimate: false,
 };
+/// Current (safe) approximation of the gas/s consumption considering
+/// EVM execution over compiled WASM.
+pub const GAS_PER_SECOND: u64 = 8_000_000;
```

Comment:
This might be high come to think of it, but we should benchmark accordingly.

Reply:
Yeah definitely should get some benchmarks before deploying this.